test/functional/specs/LibraryInfo/C2589.js

```diff
@@ -61,7 +70,11 @@ test("C2589: getLibraryInfo command returns library information.", async () => {
     orgId: "5BFE274A5F6980A50A495C08@AdobeOrg",
     thirdPartyCookiesEnabled: true,
     targetMigrationEnabled: false,
-    personalizationStorageEnabled: false
+    personalizationStorageEnabled: false,
+    autoTrackPropositionInteractions: {
+      [ADOBE_JOURNEY_OPTIMIZER]: ALWAYS,
```

Comment:
Maybe we should try and follow the `propertyEnabled` syntax for these as well too. I can see in this case that there are three states and it might be a bit more dynamic in the way these are added, but something to think about.